docs/MAINTAINING.md

```diff
@@ -0,0 +1,76 @@
+## Maintaining VTEX Toolbelt
+
+This document is for people working on VTEX as Toolbelt maintainers.
+
+### Forcing a new version
+
+Toolbelt has a way of enforcing users to use a minimum version choosed by us. This enforcement is done on the client side (can be disabled by an environment variable, check [EnvVariables](../src/lib/constants/EnvVariables.ts)) and on `vtex.builder-hub` - users using an older toolbelt version will not be allowed to link or publish. This feature is useful for enforcing users to update to the latest version, which can be important when we launch a new version with a major bug fix for example.
```

Comment:
Maybe we should explain why we do that? As someone with less context, I read that and now I'm thinking: ok, we make users have a minimum version, but why is it good for them? Should we continue doing that?

Reply:
Napoli sends me more context on Slack, and I maybe we can introduce a "Why minimumToolbeltVersion". Just three lines, won't be a huge message for readers 👀  

Suggested copy: 

"Developers have several benefits when using the updated version of our CLI. That's why we push the minimumToolbeltVersion to then: s we can ensure they won't face security bugs that may be already resolved, get our developer experiences improves, and even help us more sending use metrics."